hera_mc/tests/test_server_status.py

```diff
@@ -46,9 +46,13 @@ def test_repr(self):
                 servstat = LibServerStatus(**exp_columns)
                 class_name = 'LibServerStatus'
 
-        rep_string = ('<' + class_name + '(test_host, ' + str(exp_columns['mc_time'])
-                      + ', 0.0.0.0, ' + str(exp_columns['mc_system_timediff'])
-                      + ', 16, 20.5, 31.4, 43.2, 32.0, 46.8, 510.4, 10.4)>')
+        rep_string = ('<' + class_name + '(hostname: test_host, mc_time: '
```

Comment:
Why is this in this PR?

Reply:
Because I was trying to debug things and I needed it. I can move it to another PR if you want, but I do want this change because it makes it much easier to debug.

Reply:
nope, its fine